### PR TITLE
Mjg/lilac compatibility

### PR DESCRIPTION
**Pull request by mariajgrimaldi**

### CODE REVIEW

**Comment by mariajgrimaldi** [comment]
These were the changes I made @andrey-canon

File: eox_audit_model/models.py [hunk 1]
```diff
@@ -18,13 +18,18 @@
 from django.conf import settings
 from django.contrib.sites.models import Site
 from django.db import models
-from ipware.ip import get_ip
+
```
Comment (andrey-canon) [inline eox_audit_model/models.py:21]: you can remove this blank space

**Review by andrey-canon — COMMENTED** [review]
LGTM

**Comment by andrey-canon** [comment]
https://github.com/eduNEXT/eox-audit-model/pull/9

**Pull request closed without merging**